src/bot.ts

```diff
@@ -41,6 +42,31 @@ function generateComment(claims: BotClaimData[]): string {
   return comment;
 }
 
+function generateIssueComment(claims: BotClaimData[]): string {
```

Comment:
thoughts on moving these comment generation functions `generateIssueComment` and `generateComment` to their own  `comments.ts` file.